src/views/events.html.twig

```diff
@@ -0,0 +1,41 @@
+{% extends layout %}
+{% block meta %}
+  <title>Events - PHP Surrey</title>
+{% endblock meta %}
+
+{% block content %}
+<h2>Events</h2>
+
+<p>Signup to <a href="http://www.eventbrite.co.uk/e/php-surrey-inaugural-september-meetup-registration-17870907365?aff=www">PHP Surrey Inaugural September Meetup on EventBrite</a></p>
+
+<script type="application/ld+json">
```

Comment:
So we can have the title bar change as users move between pages. Also the `application/ld+json` is structured data (So if you search for PHP Surrey you can see the upcoming events in the google results).